src/pages/en/guides/imports.md

```diff
@@ -17,9 +17,37 @@ Astro uses Vite as its internal build system. Vite provides Astro with built-in
 - Markdown (`.md`)
 - WASM (`.wasm`)
 
-Any files in your `public/` directory are copied into the final build, untouched by Vite or Astro. The following applies to files in your `src/` directory, which Astro is ultimately responsible for.
+## `public/`
+Any files in your `public/` directory are copied into the final build, untouched by Vite or Astro. 
 
-## JavaScript & ESM
+Reference them in your Astro component template via absolute URL paths, from the `public/` directory.
+
+```
+├── public/
+│   ├── images/
+|   |   └── photo1.jpg
+│   ├── styles/
+|   |   └── global.css
+|   └── utils.js
+```
+
+```astro
+---
+// Astro component script
+---
+<head><link rel="stylesheet" href="/styles/global.css"></head>
+<body>
+  <img src="/images/photo1.jpg"> 
+
+  <script type="module" src="/nav-utils.js"></script>
+</body>
+
+```
+
+## `src/`
+ The following applies to files in your `src/` directory, which Astro is ultimately responsible for.
```

Comment:
```suggestion
 The following applies to files that are located within your `src/` directory, which Astro is wholly responsible for.
```

Reply:
Actually, I like *neither* of those "Astro is responsible for" options.  :)

What about changing that to something descriptive:

"The following applies to files located within your `src/` directory, which are processed by Astro."  ??